Demo/AB/PropertyABColor.cpp

```diff
@@ -27,3 +27,17 @@ void QtnPropertyABColor::setClickHandler(
 {
 	QObject::connect(this, &QtnPropertyABColor::click, clickHandler);
 }
+
+bool QtnPropertyMyColor::fromActualValue(const ValueTypeStore& actualValue, OldValueType& oldValue) const
```

Comment:
`OldValueType` name is a bit confusing. When converting back and forth it is hard to choose what is 'old' and what is 'new'. I'd suggest a pair of `ActualType` and `EditableType` or something similar.